tests/components/rfxtrx/test_config_flow.py

```diff
@@ -613,6 +613,73 @@ async def test_options_add_remove_device(hass):
     assert not state
 
 
+async def test_options_remove_multiple_devices(hass):
+    """Test we can add a device."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            "host": None,
+            "port": None,
+            "device": "/dev/tty123",
+            "debug": False,
+            "automatic_add": False,
+            "devices": {
+                "0b1100cd0213c7f230010f71": {"device_id": ["11", "0", "213c7f2:48"]},
+                "0b1100100118cdea02010f70": {"device_id": ["11", "0", "118cdea:2"]},
+                "0b1100101118cdea02010f70": {"device_id": ["11", "0", "1118cdea:2"]},
+            },
+        },
+        unique_id=DOMAIN,
+    )
+    entry.add_to_hass(hass)
+
+    await hass.config_entries.async_setup(entry.entry_id)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("binary_sensor.ac_213c7f2_48")
+    assert state
+    state = hass.states.get("binary_sensor.ac_118cdea_2")
+    assert state
+    state = hass.states.get("binary_sensor.ac_1118cdea_2")
+    assert state
+
+    device_registry = await async_get_registry(hass)
+    device_entries = async_entries_for_config_entry(device_registry, entry.entry_id)
+
+    assert len(device_entries) == 3
+
+    result = await hass.config_entries.options.async_init(entry.entry_id)
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "prompt_options"
+
+    result = await hass.config_entries.options.async_configure(
+        result["flow_id"],
+        user_input={
+            "debug": False,
+            "automatic_add": False,
+            "remove_device": [
+                device_entries[0].id,
+                device_entries[1].id,
+                device_entries[2].id,
```

Comment:
Maybe test leaving one device, and just remove two out of three?

Reply:
Probably better, I skipped that initially because of complexity (finding out which event code belong to which device entry) but having the device id's I think this is not that difficult after all. Will give it a try.

Reply:
Turned out not to be that much work after all.